velox/type/Timestamp.h

```diff
@@ -186,7 +190,7 @@ void parseTo(folly::StringPiece in, ::facebook::velox::Timestamp& out);
 
 template <typename T>
 void toAppend(const ::facebook::velox::Timestamp& value, T* result) {
-  // TODO Implement
+  result->append(value.toString(Timestamp::Precision::kMilliseconds));
```

Comment:
> I can fix the repeated double in next forced commit

Would you make a separate PR for this? Thanks.

Reply:
> I dont see how it will fit in CastExpr. But, we can add special path in Converter(code snippet in your comment) where 

Converter also seems rather generic, not Presto-specific or CAST specific.

Perhaps, we can modify applyCastKernel in CastExpr to add a special case:

```
if constexpr (CppToType<To>::typeKind == TypeKind::VARCHAR) {
  if constexpr (CppToType<From>::typeKind == TypeKind::TIMESTAMP) {
   ....
```

Reply:
We can do it in CastExpr too. 
Converter has cast methods containing logic for casting and I think handling all the scenarios in converter keeps the logic in one place - e.g. VARCHAR conversion to delegated to generic method which converts from T to VARCHAR and that method can handle type specific behaviors and constraints ?

Reply:
Looking at pattern, we can define type specific overrides. Can you please check latest implementation
https://github.com/facebookincubator/velox/pull/4377/files#diff-4c37e417ad24ae2e758b87ca1be0013a0e106e9e594b19eb9bbc9295a7c31620